x-pack/plugins/threat_intelligence/public/modules/cases/components/add_to_existing_case/add_to_existing_case.tsx

```diff
@@ -57,9 +59,11 @@ export const AddToExistingCase: VFC<AddToExistingCaseProps> = ({
     attachmentMetadata
   );
 
-  // disable the item if there isn't an indicator name
+  // disable the item if there is no indicator name or if the user doesn't have the right permission
   // in the case's attachment, the indicator name is the link to open the flyout
-  const disabled: boolean = attachmentMetadata.indicatorName === EMPTY_VALUE;
+  const invalidIndicatorName: boolean = attachmentMetadata.indicatorName === EMPTY_VALUE;
+  const hasUpdatePermission: boolean = permissions.read && permissions.push;
```

Comment:
@cnasikas same here, not entirely sure these are the 2 permissions required for a add to existing case feature

Reply:
Hey! You need `create` and `update` to add an attachment to a case (existing or new)